docs/usage/key-concepts/how-it-works.md

```diff
@@ -0,0 +1,152 @@
+---
+title: How it works ?
+description: Learn all about Renovate's workflow
+---
+
+# Introduction
+
+Renovate works by searching for all the dependencies in your project and then look for updates for each.
+
+To be compatible with the variety of dependency-naming and versioning conventions, Renovate has defined modules for each known convention, and you can define your own too.
+
+Please add comments to the [issue#25091](https://github.com/renovatebot/renovate/issues/25091) if you wish to see a part (better) shown in the graph below
```

Comment:
```suggestion
Please comment in [Renovate issue 25091](https://github.com/renovatebot/renovate/issues/25091) if you wish to see a part (better) shown in the graph below.
```

You're closing the linked issue with this PR. After some time, _closed_ issues are automatically _locked_ by a bot:

https://github.com/renovatebot/renovate/blob/0ecc1332b3c4260dd27bc573fb08475d737fa5b2/.github/workflows/lock.yml#L19-L28

When a issue is locked, only maintainers can comment on them.

Maybe you should create a new discussion, and link to that. I think we keep discussions open forever?

https://github.com/renovatebot/renovate/blob/main/.github/workflows/lock.yml

Reply:
I'll create enother discussion you're right